apps/roboshield/src/theme/index.ts

```diff
@@ -574,7 +574,7 @@ deepmerge(
       },
     },
   } as ThemeOptions,
-  { clone: false },
+  { clone: false }
```

Comment:
@m453h check your linter

Reply:
cool fixed it , its seems my linter was misleading me 🤔
<img width="171" alt="Screenshot 2024-07-12 at 10 03 14" src="https://github.com/user-attachments/assets/3948f509-054e-444e-b93c-688ba21248dc">